examples/production/values.yaml

```diff
@@ -37,7 +21,43 @@ spire-server:
       drop: [ALL]
     seccompProfile:
       type: RuntimeDefault
+
+  topologySpreadConstraints:
+    - maxSkew: 1
+      topologyKey: topology.kubernetes.io/zone
```

Comment:
I don't think a lot of clusters have zone defined, and DoNotSchedule together might break things?